src/components/utility/Toolttip.tsx

```diff
@@ -48,9 +50,9 @@ const Tooltip = ({
    };
 
    return (
-      <div className='relative inline-block' onClick={onClick}>
+      <div className='relative' onClick={onClick}>
          <div
-            className='inline-block'
+            className={`inline-block ${containerClassName}`}
```

Comment:
when would you ever want this container not to have a full width of the parent div?

Reply:
good point